install/gce-installation-guide.md

```diff
@@ -0,0 +1,119 @@
+# Installing Kata Containers on Google Compute Engine
+
+Kata Containers on Google Compute Engine (GCE) makes used of [nested virtualization](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances). Most of the installation procedure is identical to that for Kata on your preferred distribution, but enabling nested virtualization currently requires extra steps on GCE. This guide walks you through creating an image and instance with nested virtualization enabled.
```

Comment:
Worth mentioning that the `kata-runtime kata-check` checks for nested virt and will warn (but not fail) if it isn't found. See https://github.com/kata-containers/runtime/#hardware-requirements.

Reply:
Done.